docs/nightly/en/user-guide/migrate-to-greptimedb/migrate-from-influxdb.md

```diff
@@ -0,0 +1,537 @@
+
+```sql
++---------------------+----------+-----------+------+------+
+| ts                  | location | scientist | bees | ants |
++---------------------+----------+-----------+------+------+
+| 2019-08-18 00:00:00 | klamath  | anderson  |   23 | NULL |
+| 2019-08-18 00:06:00 | klamath  | anderson  |   28 | NULL |
+| 2019-08-18 00:00:00 | portland | mullen    | NULL |   30 |
+| 2019-08-18 00:06:00 | portland | mullen    | NULL |   32 |
++---------------------+----------+-----------+------+------+
+```
+
+The schema of the `census` table is as follows:
+
+```sql
++-----------+----------------------+------+------+---------+---------------+
+| Column    | Type                 | Key  | Null | Default | Semantic Type |
++-----------+----------------------+------+------+---------+---------------+
+| location  | String               | PRI  | YES  |         | TAG           |
+| scientist | String               | PRI  | YES  |         | TAG           |
+| bees      | Float64              |      | YES  |         | FIELD         |
+| ts        | TimestampNanosecond | PRI  | NO   |         | TIMESTAMP     |
+| ants      | Float64              |      | YES  |         | FIELD         |
++-----------+----------------------+------+------+---------+---------------+
+```
+
+## Database connection information
+
+Before you begin writing or querying data, it's crucial to comprehend the differences in database connection information between InfluxDB and GreptimeDB.
+
+- **Token**: The InfluxDB API token, used for authentication, aligns with the GreptimeDB authentication. When interacting with GreptimeDB using InfluxDB's client libraries or HTTP API, you can use `<greptimedb_user:greptimedb_password>` as the token.
+- **Organization**: Unlike InfluxDB, GreptimeDB does not require an organization for connection.
+- **Bucket**: In InfluxDB, a bucket serves as a container for time series data, which is equivalent to the database name in GreptimeDB.
+
+## Write data
+
+GreptimeDB is compatible with both v1 and v2 of InfluxDB's line protocol format,
+facilitating a seamless migration from InfluxDB to GreptimeDB.
+
+### HTTP API
+
+To write a measurement to GreptimeDB, you can use the following HTTP API request:
+
+::: code-group
+
+```shell [InfluxDB line protocol v2]
+curl -X POST 'http://<greptimedb-host>:4000/v1/influxdb/api/v2/write?db=<db-name>' \
+  -H 'authorization: token <greptime_user:greptimedb_password>' \
+  -d 'census,location=klamath,scientist=anderson bees=23 1566086400000000000'
+```
+
+```shell [InfluxDB line protocol v1]
+curl 'http://<greptimedb-host>:4000/v1/influxdb/write?db=<db-name>&u=<greptime_user>&p=<greptimedb_password>' \
+  -d 'census,location=klamath,scientist=anderson bees=23 1566086400000000000'
+```
+
+:::
+
+### Telegraf
+
+GreptimeDB's support for the Influxdb line protocol ensures its compatibility with Telegraf.
+To configure Telegraf, simply add `http://<greptimedb-host>:4000` URL into Telegraf configurations:
```

Comment:
Explain port `4000` is for self-hosted version. And it's `https://<greptimedb-host>` if using cloud version.

Reply:
I will modify the content for the Cloud version in the next PR.